frontend/packages/knative-plugin/integration-tests/features/eventing/eventing-events-subscription.feature

```diff
@@ -0,0 +1,35 @@
+@knative-eventing @knative
+Feature: Knative Events Subscription
+    User should be able to create Triggers for Knative Events using the Subscribe Form
+
+        Background:
+            Given user is at developer perspective
+              And user has created or selected namespace "knative-events-aut"
+
+
+        @pre-condition
+        Scenario: Create Broker and Knative Service and send event to Broker: KE-07-TC01
+            Given user updates the knative-eventing CM to enable the eventtype auto create
```

Comment:
Either you can have this as part of `When` statement or if you want to keep it in `Given` then you can change it to this:
```suggestion
            Given user has updated the knative-eventing CM to enable the eventtype auto create
```